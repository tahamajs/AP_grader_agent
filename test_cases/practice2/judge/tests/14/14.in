reserve_table 1 1
reserve_table 2 1
reserve_table 5 1
reserve_table 7 1
reserve_table 8 2
reserve_table 12 2
reserve_table 14 2
reserve_table 17 3
reserve_table 22 3
reserve_table 23 5
reserve_table 50 5
switch 2
switch 5
show_table_info 1
switch 14
switch 17
switch 22
switch 50
switch 8
switch 23
switch 22
switch 5
switch 8
switch 2
switch 1
show_table_info 2
show_table_info 3
show_table_info 4
show_table_info 5
show_table_info 6
