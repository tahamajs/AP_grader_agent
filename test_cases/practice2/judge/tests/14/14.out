Paria sits at table 1
Amir Hossein sits at table 1
Fereshte sits at table 1
Mahdi sits at table 1
Taha sits at table 2
Bob sits at table 2
Babak sits at table 2
Sipamo sits at table 3
Kiana sits at table 3
Narges sits at table 5
Sobhan sits at table 5
Amir Hossein switches seats with Mahdi!
Fereshte switches seats with Paria!
Table ID: 1
People at the table: Amir Hossein, Fereshte, Mahdi, Paria
Table remaining capacity: 0
Waiting queue length: 0
Babak switches seats with Mahdi!
Sipamo switches seats with Amir Hossein!
Kiana switches seats with Paria!
Sobhan switches seats with Fereshte!
Taha switches seats with Amir Hossein!
Narges switches seats with Paria!
Kiana switches seats with Paria!
Fereshte switches seats with Paria!
Taha switches seats with Amir Hossein!
Amir Hossein switches seats with Mahdi!
Paria switches seats with Bob!
Table ID: 2
People at the table: Amir Hossein, Paria, Taha
Table remaining capacity: 1
Waiting queue length: 0
Table ID: 3
People at the table: Mahdi, Narges
Table remaining capacity: 0
Waiting queue length: 0
Table ID: 4
People at the table:
Table remaining capacity: 2
Waiting queue length: 0
Table ID: 5
People at the table: Bob, Kiana
Table remaining capacity: 1
Waiting queue length: 0
Table ID: 6
People at the table:
Table remaining capacity: 3
Waiting queue length: 0
